src/scheduler/resv_info.c

```diff
@@ -132,14 +133,15 @@ stat_resvs(int pbs_sd)
  *  reservation retains its currently allocated resources, such that no other
  *  requests make use of the same resources.
  *
- *	  sinfo - the server to query from
- *	  resvs - batch status of the stat'ed reservations
+ * @param[in] sinfo  - the server to query from
+ * @param[in] resvs  - batch status of the stat'ed reservations
+ * @param[in] pbs_sd - connection to the pbs server
  *
- *	returns an array of reservations
+ * @return    An array of reservations
  *
  */
 resource_resv **
-query_reservations(server_info *sinfo, struct batch_status *resvs)
+query_reservations(server_info *sinfo, struct batch_status *resvs, int pbs_sd)
```

Comment:
Please swap the order of arguments.  If we pass in a pbs_sd it comes first (unless policy is there, then it comes second).